test/test_operations.py

```diff
@@ -454,6 +454,51 @@ def test_cat_empty_tensor(self):
     x_cat = torch.cat([x, empty_tensor_xla], 0)
     self.assertEqual(t_cat.data, x_cat.data.cpu())
 
+  def test_nan_to_num_in_place(self):
+    t = torch.tensor([float('nan'), float('nan'), -float('nan'), 3.14])
```

Comment:
Can you follow other test cases that use `self.runAtenTest()`? i.e. have a lambda as the test case and let the runner handles the device movement and comparison?

Reply:
Updated